menas/ui/css/style.css

```
@@ -127,3 +127,7 @@ h5[id$=datasetDetailView--scheduleTimingTitle] {
   padding: 8px 2px;
   margin-top: 4px;
 }
+
+.italic{
+  font-style: italic;
+}
```

And I have a side reason for above 😉 

```suggestion
}

div.sapMObjStatusInformation.missingPropertyValue span.sapMObjStatusText{
  color: silver;
}

To get "rid of" the blue.
```